### PR TITLE
LPS-80726 Organizations APIO API - Read Review

### PR DESCRIPTION
**Pull request by alejandrohdezma**

### CODE REVIEW

**Comment by ahdezma** [comment]
cc @eperezg

**Comment by ahdezma** [comment]
ci:test

**Comment by pyoo47** [comment]
ci:test

**Comment by ahdezma** [comment]
ci:reopen

**Comment by ahdezma** [comment]
ci:test:relevant

**Comment by brianchandotcom** [comment]
Merged. Thank you.
View total diff: https://github.com/brianchandotcom/liferay-portal/compare/fbe56dc362...5dbf691b52

**Comment by brianchandotcom** [comment]
@ahdezma I merged this, but I need you to fix a lot of packages in headless-api**

1.)

headless-api/**ItemResource.java needs to match

apio.internal.architect.resource vs. apio.internal.resource to match com.liferay.apio.architect.resource.ItemResource

Same for other resources.

AggregateRatingIdentifier needs to go from com.liferay.aggregate.rating.apio.identifier to com.liferay.aggregate.rating.apio.architect.identifier to match com.liferay.apio.architect.identifier.Identifier

Basically, you're missing the "architect" word in a bunch of packages.

2.) Same for FolderForm, 

3.) You have a bunch of *Router in .resource when they should be in .architect.router.

4.) Apply fccdc2af100b017ca1b92b88f232ca4fa25439ba to all of headless-apio* if this is correct.

Please fix this in your next pull to me. Thank you.
